apps/glusterfs/volume_entry_create.go

```diff
@@ -11,11 +11,12 @@ package glusterfs
 
 import (
 	"fmt"
+	"strings"
+
 	"github.com/boltdb/bolt"
 	"github.com/heketi/heketi/executors"
 	"github.com/heketi/heketi/pkg/utils"
 	"github.com/lpabon/godbc"
-	"strings"
```

Comment:
this reordering of lines (due to gofmt ???) is independent of the change of this commit and should not be sneaked in. might be a separate commit (in the same PR if you like, or in a different PR).